dash/__init__.py

```diff
@@ -39,5 +39,17 @@
     page_container,
 )
 from ._patch import Patch  # noqa: F401,E402
+from ._jupyter import jupyter_dash  # noqa: F401,E402
 
 ctx = callback_context
+
+
+def _jupyter_nbextension_paths():
```

Comment:
What is this function for?

Reply:
It's for loading the basic extension that gets the proxy settings from comm. Not really sure if it's useful, I didn't cases in testing where it was actually needed, maybe some older hub version.

Reply:
OK, we can leave it there for the time being... and if we don't find a purpose, it starts with `_` so we can remove it later without calling it breaking.